### PR TITLE
Fix Travis Test.

### PR DESCRIPTION
**Pull request by dongjoon-hyun**

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Hi, this PR fixes the travis build error.
You can check the result here, too.

https://travis-ci.org/dongjoon-hyun/PocketMine-MP

**Comment by dongjoon-hyun** [comment]
If this issue is resolved, could you merge my previous PR #3959 , too?

File: tests/TravisTest.php [hunk 1]
```diff
@@ -25,13 +25,17 @@
 	2 => ["pipe", "w"]
 ], $pipes);
 
+if(!is_resource($server)){
+	die('Fail to create process');
```
Comment (64FF00) [inline tests/TravisTest.php]: @dongjoon-hyun failed*

Comment (dongjoon-hyun) [inline tests/TravisTest.php]: Oops! Thanks! I'll fix it in an hour.

Comment (64FF00) [inline tests/TravisTest.php]: @dongjoon-hyun Tysm :+1: :smile:

**Comment by dongjoon-hyun** [comment]
PR is updated. Thank you, @64FF00 !

**Comment by 64FF00** [comment]
@dongjoon-hyun You're welcome sir :)

**Comment by dongjoon-hyun** [comment]
Thanks for merging, @PEMapModder and @64FF00 .

**Pull request merged**